csharp-advanced-topics/QueueInCSharp/Tests/QueueInCSharpUnitTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using QueueInCSharp;
+using QueueInCSharp.Models;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Tests
+{
+    public class QueueInCSharpUnitTests
+    {
+        [Fact]
+        public void whenPeekElement_thenQueueCountIsSame()
+        {
+            var queueCount = QueueOperations.OrdersCount();
+            var (firsOrder, numberOfRemainingOrders) = QueueOperations.GetOrderWithoutRemoving();
+            Assert.Equal(queueCount, numberOfRemainingOrders);
```

Comment:
Make sure to add new lines between Arrange/Act/Assert parts. Also, the method names should have capital first letters.